proxyd/rewriter.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 type RewriteContext struct {
-	latest    hexutil.Uint64
-	safe      hexutil.Uint64
-	finalized hexutil.Uint64
+	latest        hexutil.Uint64
+	safe          hexutil.Uint64
+	finalized     hexutil.Uint64
+	maxBlockRange uint64
```

Comment:
should the `maxBlockRange` be a field in the `RewriteContext` struct, or passed in as a parameter to the `RewriteTags` / `RewriteRequest` / `rewriteRange` methods?